pyproject.toml

```diff
@@ -34,6 +34,7 @@ asyncio-dgram = "1.2.0"
 click = "7.1.2"
 dnspython = "2.1.0"
 six = "1.14.0"
+tox-poetry = "^0.4.1"
```

Comment:
On my machine, I installed tox-poetry to the same place I installed tox (a pipx managed virtualenv). I'll try to add tox ond tox-poetry as a development dependency and see how that goes.